sources/packages/api/src/services/education-program-offering/education-program-offering-import-csv.service.ts

```diff
@@ -0,0 +1,227 @@
+import { Injectable } from "@nestjs/common";
+import { EducationProgram, OfferingTypes } from "../../database/entities";
+import { EducationProgramService, InstitutionLocationService } from "..";
+import {
+  SaveOfferingModel,
+  WILComponentOptions,
+} from "./education-program-offering-validation.models";
+import {
+  CSVHeaders,
+  OfferingCSVModel,
+  OfferingCSVValidationResult,
+  STUDY_BREAK_INDEX_PLACE_HOLDER,
+  YesNoOptions,
+} from "./education-program-offering-import-csv.models";
+import { plainToClass } from "class-transformer";
+import { validateSync } from "class-validator";
+import { flattenErrorMessages } from "../../utilities/class-validation";
+import { parse } from "papaparse";
+import { CustomNamedError, removeUTF8BOM } from "../../utilities";
+import { OFFERING_VALIDATION_CSV_FORMAT_ERROR } from "../../constants";
+import { InjectLogger } from "../../common";
+import { LoggerService } from "../../logger/logger.service";
+
+const MAX_STUDY_BREAKS_ENTRIES = 5;
```

Comment:
how about moving it to const file

Reply:
I left it here because it is not even exported and it has some use only for the process happening at this service.
I am not sure if moving it outside this service would bring some benefit.